### DISCUSSION TITLE
How change simple trigger value when running? AdoJob

### DISCUSSION THREAD

**Discussion by cymerman** [quartznet/quartznet#985] [General]
Hello,

How change simple trigger value when running? I dont want to update qrtz_simple_triggers table by sql (u guys alert to dont doit on your documentation)
Im using AdoJobStore with cluster and Resheduling generate error Couldn't remove trigger: Couldn't store trigger 'jobname' for 'jobname' job: 23503: insert or modification "qrtz_triggers" violate foreign key "qrtz_triggers_sched_name_fkey"

```
var trigger = TriggerBuilder.Create()
                .WithIdentity(new TriggerKey(typeof(T).FullName, appMonitoringGroup))
                .WithSimpleSchedule(schedule => schedule
                    .RepeatForever()
                    .WithMisfireHandlingInstructionIgnoreMisfires()
                    .WithIntervalInSeconds(intervalSeconds)
                )
                .StartNow()
                .Build();

                await Scheduler.RescheduleJob(new TriggerKey(typeof(T).FullName, appMonitoringGroup), trigger);

```

**Comment by cymerman** [comment]
related to https://github.com/quartznet/quartznet/issues/434 
I have job as durable but as mentioned really dont have idea how change triggers in runtime by api

**Comment by lahma** [comment]
Sounds that you have something mixed up.  `Couldn't store trigger 'jobname' for 'jobname'`, your trigger has the same name as the job?

**Reply by cymerman** [reply]
triggers have prefix JobTrigger_  so no they diffrent.  I changed real jobnames to string jobname to make easier to read

**Reply by lahma** [reply]
And your trigger builder doesn't reference a job?

**Reply by cymerman** [reply]
ok i findout it was my fail, i added id column and added as primary in script. Sorry for taking time

**Reply by lahma** [reply]
The trigger doesn't reference a job in your example, there should be a `ForJob()` call. Honestly I don't know what script you are talking about.
